tools/msys2_fetch_and_build_all_tx16s.sh

```diff
@@ -1,12 +1,147 @@
 #! /usr/bin/env bash
 
-## Bash script to show how to get EdgeTX source from GitHub (main branch),
+## Bash script to show how to get EdgeTX source from GitHub,
 ## how to build TX16S firmware, Companion, Simulator, radio simulator
 ## library and how to create an installation package.
 ## Let it run as normal user in MSYS2 MinGW 64-bit console (blue icon).
+##
+## Note: This script works properly only for branches stemming from EdgeTX v2.8 or newer
+
+# -----------------------------------------------------------------------------
+export BRANCH_NAME="main"  # main|2.8|2.9|...
+export RADIO_TYPE="tx16s"  # x9lite|x9lites|x7|x7-access|t12|tx12|tx12mk2|boxer|t8|zorro|tlite|tpro|lr3pro|xlite|xlites|x9d|x9dp|x9dp2019|x9e|x9e-hall|x10|x10-access|x12s|t16|t18|tx16s|nv14|commando8|
+
+export BUILD_OPTIONS=""
```

Comment:
Add the custom build options here:
`export BUILD_OPTIONS="-DDEFAULT_MODE=2 -DGVARS=YES -DPPM_UNIT=US -DHELI=NO -DLUA=YES"`